app/cmd/backup.go

```diff
@@ -42,6 +42,14 @@ func BackupCreateCmd() cli.Command {
 				Name:  "dest",
 				Usage: "destination of backup if driver supports, would be url like s3://bucket@region/path/ or vfs:///path/",
 			},
+			cli.StringFlag{
+				Name:  "backing-image-name",
```

Comment:
Need to understand why we have both `name` and `URL`

Reply:
`Name` will be used if the user doesn't specify the backing image when restoring this backup.
`URL` is designed to inform users which backing image the backup volume was using. Then they can check if they need to create a new backing image object before the restore.